vizro-core/docs/pages/user-guides/figure.md

```diff
@@ -105,3 +105,238 @@ To add a `Figure` to your page:
 
 If the existing figure functions from [`vizro.figures`](../API-reference/figure-callables.md) do not serve your purpose,
 there is always the option to create a [custom Figure](custom-figures.md).
+
+
+### KPI Cards
+A KPI card is a dynamic card that can display a single value. Optionally, a title, icon and reference value can be added.
+It is a common visual component to display key metrics in a dashboard. Vizro supports two pre-defined KPI card functions:
+
+- [`kpi_card`](../API-reference/figure-callables.md#kpi_card): A KPI card with a single value.
```

Comment:
These `#` anchors don't seem to work. I think they need to be `#vizro.figures.kpi_card` and similar.